From 896948b14ad26f15590269dce50ac4896284dc29 Mon Sep 17 00:00:00 2001
From: Xiao Ni <xni@redhat.com>
Date: Thu, 18 Apr 2024 18:23:21 +0800
Subject: [PATCH 62/66] tests/01raid6integ.broken can be removed

01raid6integ can be run successfully with kernel 6.9.0-rc3.
So remove 01raid6integ.broken.

Signed-off-by: Xiao Ni <xni@redhat.com>
Signed-off-by: Mariusz Tkaczyk <mariusz.tkaczyk@linux.intel.com>
---
 tests/01raid6integ.broken | 7 -------
 1 file changed, 7 deletions(-)
 delete mode 100644 tests/01raid6integ.broken

diff --git a/tests/01raid6integ.broken b/tests/01raid6integ.broken
deleted file mode 100644
index 1df735f0..00000000
--- a/tests/01raid6integ.broken
+++ /dev/null
@@ -1,7 +0,0 @@
-fails infrequently
-
-Fails about 1 in 5 with a sha mismatch:
-
-    8286c2bc045ae2cfe9f8b7ae3a898fa25db6926f /dev/md0 does not match
-    a083a0738b58caab37fd568b91b177035ded37df /dev/md0 with /dev/loop2 and
-    /dev/loop3 missing
-- 
2.41.0

